PowerEditor/src/ScitillaComponent/FindReplaceDlg.cpp

```diff
@@ -357,17 +357,17 @@ int FindReplaceDlg::saveComboHistory(int id, int maxcount, vector<generic_string
 	int count = ::SendMessage(hCombo, CB_GETCOUNT, 0, 0);
 	count = min(count, maxcount);
 
-    if (count == CB_ERR) return 0;
+	if (count == CB_ERR) return 0;
 
-    if (count)
-        strings.clear();
+	if (count)
+		strings.clear();
 

```

Comment:
I haven't changed that. The change is caused by the automatic replacement of the leading spaces with tabs that I have done in the first patch from the chain. You can review the patches one by one to get a clearer picture of the functional changes.